File: Web/Nikita12112002_NikitaGupta_1923it1159_3/GettingStarted/FibonacciSeries.cpp
#include <iostream>
using namespace std;

int main(int argc, char **argv)
{
    int n;
    cin >> n;
    
    int a=0, b=1 ,c;
    cout<<a<<endl;
    cout<<b<<endl;
    
    for(int i=2 ; i<n ;i++)
    {
        c=a+b;
        cout<<c<<endl;
        a=b;
        b=c;
    }
    
    //write your code here
    
}

File: IOT/anshulgurjar53_AnshulSinghGurjar_2024it1193_2/GettingStarted/FibonacciSeries.java
import java.util.*;
  
  public class Main{
  
  public static void main(String[] args) {
      // write your code here
      Scanner sc=new Scanner(System.in);
      int n=sc.nextInt();
      int a=0,b=1,c=0;
      System.out.println(a);
       System.out.println(b);
      for(int i=1;i<=n-2;i++)
      {
        c=a+b;
        a=b;b=c;
        System.out.println(c);
      }
      
   }
  }